composer.json

```diff
@@ -16,51 +16,52 @@
         "php": ">=8.1",
         "ext-pdo_sqlite": "*",
         "doctrine/dbal": "^3.1",
-        "doctrine/doctrine-bundle": "^2.8",
-        "doctrine/doctrine-migrations-bundle": "^3.0",
-        "doctrine/orm": "^2.11",
+        "doctrine/doctrine-bundle": "2.11.x-dev",
```

Comment:
to solve the CI warning, you could require `^2.11` here, which will install 2.11.x-dev as there is not yet a 2.11.0 release.

And there is only 1 unreleased change in 2.11.x-dev compared to the 2.10.x branch. Maybe `^2.10.3` could be used instead, as you only care about the change that is pending a 2.10.3 release.

Reply:
Good idea. Done. Thanks!